{"name": "test_invalid_business_num2", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nBacktrace:\n\tMicrosoft::Applications::Events::EventProperties::SetProperty [0x00007FF7049DA742+12658]\n\tMicrosoft::Applications::Events::EventProperty::EventProperty [0x00007FF7049729E2+747650]\n\t(No symbol) [0x00007FF70460DD2E]\n\t(No symbol) [0x00007FF70464FB49]\n\t(No symbol) [0x00007FF70464FCD0]\n\t(No symbol) [0x00007FF70468B4B7]\n\t(No symbol) [0x00007FF70466F42F]\n\t(No symbol) [0x00007FF704642791]\n\t(No symbol) [0x00007FF70468874C]\n\t(No symbol) [0x00007FF70466F1C3]\n\t(No symbol) [0x00007FF7046417D6]\n\t(No symbol) [0x00007FF7046409B5]\n\t(No symbol) [0x00007FF704641F44]\n\tMicrosoft::Applications::Events::EventProperties::SetProperty [0x00007FF7048856A0+147088]\n\t(No symbol) [0x00007FF704729306]\n\tMicrosoft::Applications::Events::EventProperty::EventProperty [0x00007FF7048C4607+33959]\n\tMicrosoft::Applications::Events::EventProperty::EventProperty [0x00007FF7048C80D5+49013]\n\tMicrosoft::Applications::Events::ILogManager::DispatchEventBroadcast [0x00007FF704BB9A93+1305267]\n\tMicrosoft::Applications::Events::EventProperty::EventProperty [0x00007FF70497AF9C+781884]\n\tMicrosoft::Applications::Events::EventProperty::EventProperty [0x00007FF7049804C4+803684]\n\tMicrosoft::Applications::Events::EventProperty::EventProperty [0x00007FF70498061C+804028]\n\tMicrosoft::Applications::Events::EventProperty::EventProperty [0x00007FF704988721+837057]\n\tBaseThreadInitThunk [0x00007FF9B63A26BD+29]\n\tRtlUserThreadStart [0x00007FF9B704A9F8+40]", "trace": "self = <tests.edgeTests.testPersonalAreaPage.TestPersonalAreaPage object at 0x000001EE6A9E2C20>\ndriver = <selenium.webdriver.edge.webdriver.WebDriver (session=\"18814156c83c400c7c7d16976914025f\")>\n\n    @allure.description(\"the test is verify, that user can't edit his profile with invalid business number\")\n    def test_invalid_business_num2(self, driver):\n>       self.sign_in_modal.sign_in()\n\ntests\\edgeTests\\testPersonalAreaPage.py:44: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nsrc\\pages\\signInModal.py:54: in sign_in\n    SignInModal.set_phone_num(self, \"0552603210\")\nsrc\\pages\\signInModal.py:13: in set_phone_num\n    CommonActions.set_locator(self, sign_in_lcs[\"phone_number_fld\"]).send_keys(phonenum)\nsrc\\utils\\commonActions.py:9: in set_locator\n    return wdw(self.driver, 5).until(ec.visibility_of_element_located(locator))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"18814156c83c400c7c7d16976914025f\")>\nmethod = <function visibility_of_element_located.<locals>._predicate at 0x000001EE6AF79900>, message = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       Backtrace:\nE       \tMicrosoft::Applications::Events::EventProperties::SetProperty [0x00007FF7049DA742+12658]\nE       \tMicrosoft::Applications::Events::EventProperty::EventProperty [0x00007FF7049729E2+747650]\nE       \t(No symbol) [0x00007FF70460DD2E]\nE       \t(No symbol) [0x00007FF70464FB49]\nE       \t(No symbol) [0x00007FF70464FCD0]\nE       \t(No symbol) [0x00007FF70468B4B7]\nE       \t(No symbol) [0x00007FF70466F42F]\nE       \t(No symbol) [0x00007FF704642791]\nE       \t(No symbol) [0x00007FF70468874C]\nE       \t(No symbol) [0x00007FF70466F1C3]\nE       \t(No symbol) [0x00007FF7046417D6]\nE       \t(No symbol) [0x00007FF7046409B5]\nE       \t(No symbol) [0x00007FF704641F44]\nE       \tMicrosoft::Applications::Events::EventProperties::SetProperty [0x00007FF7048856A0+147088]\nE       \t(No symbol) [0x00007FF704729306]\nE       \tMicrosoft::Applications::Events::EventProperty::EventProperty [0x00007FF7048C4607+33959]\nE       \tMicrosoft::Applications::Events::EventProperty::EventProperty [0x00007FF7048C80D5+49013]\nE       \tMicrosoft::Applications::Events::ILogManager::DispatchEventBroadcast [0x00007FF704BB9A93+1305267]\nE       \tMicrosoft::Applications::Events::EventProperty::EventProperty [0x00007FF70497AF9C+781884]\nE       \tMicrosoft::Applications::Events::EventProperty::EventProperty [0x00007FF7049804C4+803684]\nE       \tMicrosoft::Applications::Events::EventProperty::EventProperty [0x00007FF70498061C+804028]\nE       \tMicrosoft::Applications::Events::EventProperty::EventProperty [0x00007FF704988721+837057]\nE       \tBaseThreadInitThunk [0x00007FF9B63A26BD+29]\nE       \tRtlUserThreadStart [0x00007FF9B704A9F8+40]\n\nC:\\Users\\artem\\PycharmProjects\\POMTrado\\venv\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py:95: TimeoutException"}, "description": "the test is verify, that user can't edit his profile with invalid business number", "start": 1680536373509, "stop": 1680536379436, "uuid": "e1650ea4-484c-4b5b-b71d-6baca1813fce", "historyId": "a9f5eff4f3a4248c58631bd74542d41e", "testCaseId": "b975f07a8fe3d1a4e7a7652572f500da", "fullName": "tests.edgeTests.testPersonalAreaPage.TestPersonalAreaPage#test_invalid_business_num2", "labels": [{"name": "parentSuite", "value": "tests.edgeTests"}, {"name": "suite", "value": "testPersonalAreaPage"}, {"name": "subSuite", "value": "TestPersonalAreaPage"}, {"name": "host", "value": "krot"}, {"name": "thread", "value": "19772-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.edgeTests.testPersonalAreaPage"}]}